src/group_theory/order_of_element.lean

```diff
@@ -400,6 +400,24 @@ end
 
 end group
 
+section comm_group
+
+variables [comm_group G]
+
+/-- Elements of finite order are closed under multiplication. -/
+@[to_additive "Elements of finite order are closed under addition."]
+lemma is_of_fin_order_mul (hx : is_of_fin_order x) (hy : is_of_fin_order y) :
+  is_of_fin_order (x * y) :=
```

Comment:
```suggestion
lemma is_of_fin_order.mul (hx : is_of_fin_order x) (hy : is_of_fin_order y) :
  is_of_fin_order (x * y) :=
```
for dot notation; assuming we're in the right namespace.

Reply:
(Done in #12750, thanks)